### ISSUE TITLE
Locally Selective Combination (LSCP)

### ISSUE THREAD

**Issue by shenjian5** [yzhao062/pyod#180]
while use compare_all_models.py file,it comes with the following bug：    

raise ValueError('x and y must have length at least 2.')
ValueError: x and y must have length at least 2.

**Comment by yzhao062** [comment]
Thanks for the note. I will say just rerun it. I tried to reproduce on my end and it went through.
if possible, could you share me the code and the screenshot of the message to reproduce?  thanks

**Comment by shenjian5** [comment]
I just download the last code，and run compare_all_models.py file . Then I meet the error below： 

![error of compare_all_models.py file](https://user-images.githubusercontent.com/6245847/79678223-aadc7280-822b-11ea-9ebf-96499b9e065e.png)

**Comment by laetella** [comment]
I meet the same problem. 
![image](https://user-images.githubusercontent.com/13004786/82115757-4dd6cc80-9797-11ea-852c-a0dc7adeb064.png)
I think this error is only in some data.

**Comment by yzhao062** [comment]
Problem acknowledged. Looks like there are some update on the dependent libraries.

**Comment by yzhao062** [comment]
as shown https://github.com/yzhao062/pyod/pull/192, just change line 74 to random_state = 42
There are some underlying changes in numpy and sklearn, which cause the issue. A PR will be merged soon.

**Comment by jaketimothy** [comment]
When I run LSCP on my own data with `random_state = 42`, the issue persists.

**Comment by WLiK** [comment]
Yes, I also met this issue. For this error,  I fund that the length of local_pseudo_ground_truth and local_train_socres[:,d] were both 1 when calculating pearsonr in LSCP.py.

**Comment by SaqlainHussainShah** [comment]
Has this issue being solved? I am still facing this issue.
